.github/workflows/ci-unix-static-av2.yml

```diff
@@ -1,5 +1,10 @@
 name: CI Unix Static AV2
-on: [push, pull_request]
+  push
```

Comment:
where is this documented? Shouldn't these be under 'on:'?

Reply:
They should, just a bad copy/paste that happens to work for some reason.